### PR TITLE
Upgrade tables

### PR DESCRIPTION
**Pull request by midas-myth**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207859816703816

### CODE REVIEW

**Review by onemicky — APPROVED** [review]

**Review by onemicky — APPROVED** [review]

File: src/components/Referrals/TradersStats.tsx [hunk 1]
```diff
@@ -192,30 +196,31 @@ function TradersStats({ referralsData, traderTier, chainId, userReferralCodeStri
           </div>
         </Modal>
       </div>
-      {currentDiscountDistributions.length > 0 ? (
+      {currentDiscountDistributions.length > -1 ? (
```
Comment (onemicky) [inline src/components/Referrals/TradersStats.tsx:199]: Wondering how the `length` could be not greater than -1?

File: src/components/Synthetics/SelectorBase/SelectorBase.tsx [hunk 2]
```diff
@@ -81,53 +82,66 @@ export function SelectorBaseMobileButton(
...
-    return props.children;
-  }, [props.children, props.message]);
-
   if (props.disabled && props.disabledMessage) {
     return (
       <TooltipWithPortal
-        as="tr"
-        className={cx("SelectorBaseUtils-row", props.className)}
+        as={TableTr}
+        className={cx("SelectorBaseUtils-row ", props.className)}
```
Comment (onemicky) [inline src/components/Synthetics/SelectorBase/SelectorBase.tsx:97]: Unneeded space

**Comment by onemicky** [comment]
Let's also resolve the conflict

File: src/components/Referrals/TradersStats.tsx (hunk 1 above)
Comment (midas-myth) [reply src/components/Referrals/TradersStats.tsx]: very strange indeed, set to 0 as it was

File: src/components/Synthetics/SelectorBase/SelectorBase.tsx (hunk 2 above)
Comment (midas-myth) [reply src/components/Synthetics/SelectorBase/SelectorBase.tsx]: removed, also removed the same issue sevelar lines below

**Review by onemicky — DISMISSED** [review]

**Pull request merged**